sensor_msgs/Image im
string[] types
cvRect[] rects
---
classificationInfo[] results
